src/python/aim/_sdk/collections.py

```diff
@@ -131,6 +141,23 @@ def count(self):
         # more optimal implementation
         return sum(1 for _ in self.__iter_meta__())
 
+    def delete(self):
+        from aim._sdk import Repo
+        from aim._sdk import Container
+        from aim._sdk.lock_manager import RunLockingError
+        repo = Repo.active_repo()
```

Comment:
If we have multiple open repos, this line would fail. The `repo` instance should be available in the collecrtion's `query_context` dict.